files/grest/cache_queries/epoch_info_cache_setup.sql

```diff
@@ -0,0 +1,204 @@
+DROP TABLE IF EXISTS grest.epoch_info_cache;
+
+DROP TABLE IF EXISTS temp_active_stake;
```

Comment:
Unless really needed, would be good to avoid writing to public schema, same for other tables below